src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20221003123755_EES3751_AddPermalink.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES3751_AddPermalink : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Permalinks",
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(type: "uniqueidentifier", nullable: false),
+                    PublicationTitle = table.Column<string>(type: "nvarchar(max)", nullable: false),
+                    DataSetTitle = table.Column<string>(type: "nvarchar(max)", nullable: false),
+                    ReleaseId = table.Column<Guid>(type: "uniqueidentifier", nullable: true),
+                    SubjectId = table.Column<Guid>(type: "uniqueidentifier", nullable: false),
```

Comment:
When read replica goes out, we can revisit these, as some of them can be made non-nullable if we choose to.

Reply:
Yeah, leave as nullable for now then :+1: